src/DynamoCore/Extensions/ReadyParams.cs

```diff
@@ -60,6 +61,14 @@ public virtual ICommandExecutive CommandExecutive
         }
 
         /// <summary>
+        /// Return logger for sending information to Dynamo console
+        /// </summary>
+        public ILogger Logger
+        {
+            get { return dynamoModel.Logger; }
```

Comment:
We have already exposed a logging facility via `ICommandExecutive` that inherits from `ILogSource`. All implementations of `ICommandExecutive`, need to implement the `MessageLogged` event. Currently, `ExtensionCommandExecutive` and `ViewExtensionCommandExecutive` implement this with a simple call to `DynamoModel.Logger.Log("message")`. Using this framework an appropriate message can be logged whenever a recordable command is executed in the extension.